app/airq/commands/base.py

```diff
@@ -2,6 +2,8 @@
 import re
 import typing
 
+from flask_babel import gettext
```

Comment:
unused import?

Reply:
I added these when I added gettext everywhere, but then removed gettext when I was trying to get a single string to be translated. I can remove this instance and other instances in the other files if you want, but we will be adding gettext in this file in the future.

Reply:
ok cool to keep it in then